### PR TITLE
Prepare for release  2.15.3

### PR DESCRIPTION
**Pull request by makarandp0**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,21 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x will End of Life on September 8th, 2021.** Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) to plan your migration to the latest 2.x version. Support for the 1.x version ended on December 4th, 2020.
 
+2.15.3 (July 28, 2021)
+======================
+
+Bug Fixes
+---------
+Chrome 92 [started enforcing](https://chromium-review.googlesource.com/c/chromium/src/+/2816118) limit on number of WebMediaPlayers. This blocks creation of  WebMediaPlayers once there are already 75 (desktop) or 40 (mobile) players. Fixed a related bug where the SDK was not cleaning up internally maintained media elements.
+Please ensure that your application cleans up media elements as well after they are detached (VIDEO-6336).
```
Comment (charliesantos) [inline CHANGELOG.md:11]: We should get another pair of eyes on this changelog. @anna-vasilko @donaltoomey @syerrapragada 

I think we need to reword it such that this change is not really fixing the chrome 92 issue. It is only helping prevent that issue on certain scenarios like reconnecting or republishing. What do you think of this? Feel free to change it.

> Fixed a bug where the SDK was not cleaning up internally maintained media elements. This causes memory leaks on certain use cases such as reconnecting or republishing to a room.
>
> Additionally, Chrome 92 [started enforcing](https://chromium-review.googlesource.com/c/chromium/src/+/2816118) limit on number of WebMediaPlayers. This blocks creation of  WebMediaPlayers once the limit is reached - 75 for desktop and 40 for mobile. This SDK update will help prevent running into this limit issue on use cases such as reconnecting or republishing to a room. Please ensure that your application cleans up media elements as well after they are detached.
>
>```js
>  const elements = track.detach();
>  elements.forEach(el => {
>    el.remove();
>    el.srcObject = null;
>  });
>```
>
> Please be aware that your application may still run into the Chrome's WebMediaPlayers limit for large rooms where participants exceeds this limit.

I think we need to add information on resolutions we're investigating for large rooms.

**Review by charliesantos — CHANGES_REQUESTED** [review]

**Review by donaltoomey — COMMENTED** [review]
Is this fix the only change? Seems like there are other changes too, e.g. the isSupported flag update

File: CHANGELOG.md (hunk 1 above)
Comment (makarandp0) [reply CHANGELOG.md:11]: Thanks @charliesantos, I like your text. I will update.

**Comment by makarandp0** [comment]
> Is this fix the only change? Seems like there are other changes too, e.g. the isSupported flag update

@donaltoomey, Yes, the fix is the only change (as seen in files changed) - Somehow the older commits are showing up, which I suspect because somewhere we "squash merged" into main instead of merge. It should get fixed after this.

**Comment by donaltoomey** [comment]
Ok, that was it.
LGTM

**Review by donaltoomey — APPROVED** [review]
LGTM

**Review by charliesantos — APPROVED** [review]
Looks good! Did you still want to link to this chrome issue? https://chromium-review.googlesource.com/c/chromium/src/+/2816118

**Pull request merged**